db/initdb.d/fosslight_create.sql

```diff
@@ -1449,13 +1482,15 @@ DELETE FROM `PRE_OSS_COMPONENTS`;
 -- 테이블 fosslight.PRE_OSS_COMPONENTS_LICENSE 구조 내보내기
 DROP TABLE IF EXISTS `PRE_OSS_COMPONENTS_LICENSE`;
 CREATE TABLE IF NOT EXISTS `PRE_OSS_COMPONENTS_LICENSE` (
+  `COMPONENT_ID2` int(11) NOT NULL AUTO_INCREMENT,

```

Comment:
What does COMPONENT_ID2 mean? What's the difference with COMPONENT_ID?

Reply:
@riyenas0925 

Currently, the COMPONENT_ID of self-check is a combination of (REFERENCE_ID + REFERENCE_DIV + COMPONENET_IDX). You can query PRE_OSS_COMPONENTS_LICENSE in multiple mapper functions only if you have a unique id that is AUTO_INCREMENT. So I added COMPONENT_ID2(Same as COMPONENT_ID in OSS_COMPONENTS_LICENSE).